### PR TITLE
fix: clone app action templates correctly

### PR DESCRIPTION
**Pull request by martin3walker**

### CODE REVIEW

File: packages/contentful--create-contentful-app/src/includeAppAction.ts [hunk 1]
```diff
@@ -1,62 +1,61 @@
...
+import degit from 'degit';
+import { success, highlight } from './logger';
 
-export function cloneAppAction(templateIsTypescript: boolean, destination: string) {
-  const actionPath = resolve(
-    templateIsTypescript
-      ? 'src/app-actions/typescript/index.ts'
-      : 'src/app-actions/javascript/index.js'
-  );
+export async function cloneAppAction(templateIsTypescript: boolean, destination: string) {
```
Comment (colomolo) [inline packages/contentful--create-contentful-app/src/includeAppAction.ts:8]: Strange that non-primary boolean param goes first. Should be second and non-mandatory, I think (with fallback to JS IMO).

File: packages/contentful--create-contentful-app/src/includeAppAction.ts [hunk 2]
```diff
@@ -1,62 +1,61 @@
...
+  try {
+    console.log(highlight('Cloning hosted app action.'));
+    // Clone the app actions template to the created directory under the folder 'actions'
+    const templateSource = `contentful/apps/examples/hosted-app-action-templates/${
```
Comment (colomolo) [inline packages/contentful--create-contentful-app/src/includeAppAction.ts:12]:
```suggestion
    const templateSource = path.join('contentful/apps/examples/hosted-app-action-templates', templateIsTypescript ? 'typescript' : 'javascript');
```

Comment (martin3walker) [reply packages/contentful--create-contentful-app/src/includeAppAction.ts:12]: Is this just for readability, or is there something else that it does?

File: packages/contentful--create-contentful-app/src/includeAppAction.ts (hunk 1 above)
Comment (martin3walker) [reply packages/contentful--create-contentful-app/src/includeAppAction.ts:8]: I can switch the order, but I think it needs to be mandatory. If we're not explicit we run the risk of cloning js files into a typescript repo, which will cause problems for users. Wdyt?

Comment (colomolo) [reply packages/contentful--create-contentful-app/src/includeAppAction.ts:8]: Maybe it's better to switch it from boolean to `'typescript' | 'javascript'` type to support other templates in the future?

Comment (colomolo) [reply packages/contentful--create-contentful-app/src/includeAppAction.ts:8]: > If we're not explicit we run the risk of cloning js files into a typescript repo, which will cause problems for users.

Yes, makes sense

**Review by colomolo — APPROVED** [review]

File: packages/contentful--create-contentful-app/src/includeAppAction.ts (hunk 2 above)
Comment (martin3walker) [reply packages/contentful--create-contentful-app/src/includeAppAction.ts]: Nice, thanks!

**Pull request merged**